src/cast_core/test/CrudActor_test.cpp

```diff
@@ -45,8 +45,6 @@ TEST_CASE_METHOD(MongoTestFixture,
       - Name: CrudActor
         Type: CrudActor
         Database: mydb
-        RetryStrategy:
```

Comment:
There will be merge conflicts with #219. I can take of them. (Just need to bulk remove lines containing `RetryStrategy` and `ThrowOnFailure` in this file.

Reply:
May the odds be ever in your favor
(But if you end up merging first that's fine it's an easy conflict)

Reply:
Well it was pretty much the easiest merge conflict. `RetryStategy` doesn't appear at all in the Yaml test so I just had to remove it once from the runner test CPP file.